src/ooxml/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -649,12 +649,11 @@ public PackagePart getPart(PackagePartName partName) {
 	 */
```

Comment:
Yep, I've been running a version of this in production since Jan. I rebased before opening this pull request yesterday